src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/BeneficialOwnerChangeService.java

```diff
@@ -225,6 +225,10 @@ private BeneficialOwnerChange<CorporateBeneficialOwnerPsc> convertBeneficialOwne
         Identification::getRegistrationNumber,
         CompanyIdentification::setRegistrationNumber);
 
+    if (!beneficialOwnerCorporateDto.getTrustIds().isEmpty()) {
```

Comment:
Maybe always just set - which would be set to null if getTrustIds is empty.

Reply:
yep i have added this just was waiting to push with unit tests